### DISCUSSION TITLE
"Help > What's New" tab inquiry

### DISCUSSION THREAD

**Discussion by thatsaksyguy** [obsproject/obs-studio#9791] [General]
Hey all!

First time ever attempting to contribute in some way to an open source project so apologies if this is a dumb question, but is the "What's New" tab in the app supposed to come up with nothing right now? I keep clicking on it and nothing pops up. Is this intentional?

Context: I'm currently trying to find some way to contribute to a project for my Open Source class. I just downloaded OBS studio from the website and as I was clicking around for a minute this was the first thing that seemed a bit odd to me. 

I thought I would ask just in case it was intentional. Which, if it is, then no worries, I will look for some other way to contribute. However, my thinking is if there is a "What's New" tab then *something* should still pop up, and if not then maybe it shouldn't be there? I tried looking around for additional documentation for it, as well as discussions or issues, but didn't see anything (closest I could find is a ticket for the beta around a year ago: https://github.com/obsproject/obs-studio/issues/7158) so I figured I would create a discussion post first before creating an actual issue ticket or something. 

Thanks!

**Comment by Fenrirthviti** [comment]
"What's New" is a bit of a misleading and unfortunate name, as it's really more of an active notification system than something that is expected to be on all the time. We use it for notices and other updates on occasion, but checking for updates and there not being anything right now is very much expected, yes. There is no bug to fix there.

**Reply by thatsaksyguy** [reply]
I guess that makes sense, thank you for clarifying that! It may be just because I'm used to something like Steam's "Update News" that comes up with a separate window for games that have been updated. If what you described is the intended purpose of it, do you think I could put in a suggestion somewhere to change the name of the tab to something more akin to "Notifications" or "Updates" instead - maybe with a number or symbol to indicate if there are any/how many before clicking - to make its purpose a bit more clear?
